diff --git a/dist/hooks.js b/dist/hooks.js
index 508e9057b5e9a1cd354a2b4308e5ede3447c5d67..ec856da10e1e426a33ca8452fc9c0b7200e7f712 100644
--- a/dist/hooks.js
+++ b/dist/hooks.js
@@ -59,7 +59,7 @@ export function experimental_useInboxSender(inboxOwnerID) {
         throw new Error("useInboxSender can't be used in a JazzProvider with auth === 'guest'.");
     }
     const me = context.me;
-    const inboxRef = useRef();
+    const inboxRef = useRef(undefined);
     const sendMessage = useCallback(async (message) => {
         if (!inboxOwnerID)
             throw new Error("Inbox owner ID is required");
diff --git a/src/hooks.ts b/src/hooks.ts
index 070ebb3c42f99138c399eb27d1333b7180ee6b33..2769b8a31932a9affa1468793cc80357e5143e23 100644
--- a/src/hooks.ts
+++ b/src/hooks.ts
@@ -127,7 +127,7 @@ export function experimental_useInboxSender<
   }
 
   const me = context.me;
-  const inboxRef = useRef<Promise<InboxSender<I, O>> | undefined>();
+  const inboxRef = useRef<Promise<InboxSender<I, O>> | undefined>(undefined);
 
   const sendMessage = useCallback(
     async (message: I) => {
